---
title: "Wastewater Surveillance for SARS-CoV-2 Data Analysis Project"
subtitle: ""
author: Leah Lariscy
date: "`r Sys.Date()`"
format:
  html:
    toc: false
    number-sections: true
    number-depth: 2
    highlight-style: github
bibliography: ../citation.bib
csl: ../apa.csl
---

```{r, echo=FALSE, message=FALSE}
# load packages
library(here)
library(knitr)
library(tidyverse)
library(skimr)
```

# Abstract

Infection by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), causing COVID-19, is followed by the shedding of viral particles by multiple excretory functions, including stool and urine production. Subsequently, these viral particles can be detected in wastewater influent via RNA extraction, followed by Reverse Transcriptase Quantitative Polymerase Chain Reaction (RT-qPCR) utilizing SARS-CoV-2 specific primers. Wastewater-based detection methods have been utilized across the globe as an independent and parallel indicator of SARS-CoV-2 viral prevalence and, depending on community-specific factors, as a leading indicator of clinical case trends. WBE methods may also become a primary tool in understanding transmission dynamics as case under-reporting increases.

{{< pagebreak >}}

# Introduction

## General Background Information

Paragraph 1: Impact of COVID-19 on public health, clinical case under-reporting

Paragraph 2: Wastewater surveillance is possible because of excretory functions and stability of viral particles in wastewater, surveillance prevalence and correlations with clinical case data

Paragraph 3: Surveillance program background. Aims, County-level parallel analysis of wastewater surveillance data and reported clinical cases

## Description of data and data source

### SARS-CoV-2 wastewater surveillance data

Sample collection for wastewater surveillance began June 30, 2020 and occurred twice weekly (excluding major holidays) through December 21, 2022. Raw wastewater influent was collected from three Wastewater Treatment Plants in Athens-Clarke County and viral RNA was extracted. RT-qPCR was used to calculate the cycle threshold (Ct) value of each reaction, which is used to estimate SARS-CoV-2 copies per liter of wastewater in the data cleaning process (Section 3.2).

### Wastewater treatment plant data

Wastewater influent flow data, including millions of gallons per day (MGD) and total suspended solids (TSS), were collected for each corresponding sampling date.

### COVID-19 clinical data

COVID-19 clinical data were obtained from the Georgia Department of Public Health website [@COVID19StatusReport] which includes COVID-19 reported clinical case data for the state of Georgia. These data include reported cases, demographics, comorbidities, deaths, and more. However, not all data is necessary for analysis and will be subset to include only symptom onset, positive case, and administered test data for Clarke County. (Section 3.2).

## Hypotheses to be addressed

This study aims to conduct a parallel analysis between SARS-CoV-2 wastewater surveillance data and reported COVID-19 clinical case data in Athens-Clarke County, Georgia. It is expected that wastewater quantifications and reported case data will have a positive linear relationship. It is also hypothesized that certain wastewater-clinical relationships will be stronger than others, see Table 1 for variables of interest.

{{< pagebreak >}}

# Methods

## Data acquisition

### Wastewater surveillance data

24-hour composite raw wastewater influent samples were collected twice-weekly from three treatment facilities in Clarke County and stored at 4¬∞C until ready for extraction. 280 ùúáL of wastewater in replicates (n=6) was aliquoted and direct RNA extraction (Zymo Research, R2042) was performed on the same day as sample collection.Each extraction replicate was reverse transcribed and amplified using RT-qPCR and a SARS-CoV-2 specific primer-probe panel (IDT, 10006713), done in technical replicates (n=3). As part of the qPCR panel, two gene targets on the N-gene of the SARS-CoV-2 genome were used for amplification. The N1 and N2 gene targets were chosen due to their conservation across variants and act as universal detectors of the virus \[\@\]. Cycle threshold (Ct) values were then calculated for each PCR reaction, which were then used to estimate SARS-CoV-2 viral load (See Section 3.2 for more detail).

### Wastewater treatment plant data

For each treatment plant and collection date, MGD and TSS was shared by plant operators directly with our team and compiled throughout the study period.

### COVID-19 clinical data

Reported case and testing data were downloaded from Georgia Department of Public Health's COVID-19 Status Report Page in January 2023.

## Data import and cleaning

### Calculating wastewater viral load

Amplification standard curve equations where generated from serially diluted synthetic controls (Supplementary Table). This was done for N1 and N2 gene targets for both surveillance years because RT-qPCR methods were altered after the first year. Viral copies per microliter (cp/uL) of each PCR reaction was estimated by transforming the Ct value using the appropriate standard curve equation. For reactions with no detection, concentration was set to the theoretical limit of detection (0.004 cp/uL). Data sets for each standard curve was transformed separately before being combined. After combining all qPCR data, copies per liter (cp/L) of wastewater was calculated for each technical replicate using Equation 1. Average cp/L of wastewater was determined by taking the mean of all technical replicates per sample per gene target.

INSERT EQUATION 1

WWTP data was then transformed by converting the flow rate from millions of gallons per day to liters per day. qPCR data was then combined with WWTP data and total viral load per sample was calculated by multiplying average copies per liter by the flow rate. Average viral load for each sample was then determined by taking the mean of the viral load for both N1 and N2 gene targets. County-level daily viral load was calculated by taking the sum of the average viral load from each of the three WWTP samples.

### Calculating wastewater qPCR assay positivity

qPCR assay positivity was determined for each collection date by finding the proportion of assays that detected SARS-CoV-2 out of the total number of assays performed. This includes all biological and technical replicate assays that were performed for a given sampling date. County-level assay positivity data was then combined with county-level viral load data.

### Adding COVID-19 clinical data

Of the downloaded Georgia DPH data, the data set containing positive case counts was retained. Of these data, daily reported case counts and case seven-day moving average were kept for analysis. These data were then subset to include only data from Clarke County collected during the surveillance period and combined with wastewater data.

## Statistical analysis

Through exploration of time series and wastewater-clinical correlations, prominent relationships were identified for further analysis. Wastewater variables were used to train linear regression models to predict clinical observations. Model performance metrics (RSQ and RMSE) were assessed against each other and against a null model. Cross-validation was used to validate performances. The best model was then determined and used to predict values in a new set of data.

{{< pagebreak >}}

# Results

## Time-series analysis

## Correlation analysis

Two wastewater and two clinical case metrics were identified as features of interest during exploratory analysis steps where stronger wastewater-clinical correlations were observed between these variables than in others. These include wastewater viral load, wastewater assay positivity rate, clinical case seven-day moving average, and test administration positivity rate (Table 1).

```{r, echo=FALSE, message=FALSE}

readRDS(here("results/manuscript_table1.rds"))
```

## Model analysis

```{r, echo=FALSE, fig.cap="Observed Clinical Cases vs Wastewater Predictions"}
include_graphics(here("figures/modeling/assaypos_7dma.png"))
```

{{< pagebreak >}}

# Discussion

## Summary and Interpretation

*Summarize what you did, what you found and what it means.*

## Strengths and Limitations

*Discuss what you perceive as strengths and limitations of your analysis.*

## Conclusions

*What are the main take-home messages?*

{{< pagebreak >}}

# References
