<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.313">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Leah Lariscy">
<meta name="dcterms.date" content="2023-04-25">

<title>Wastewater Surveillance for SARS-CoV-2 Data Analysis Project</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="Manuscript_files/libs/clipboard/clipboard.min.js"></script>
<script src="Manuscript_files/libs/quarto-html/quarto.js"></script>
<script src="Manuscript_files/libs/quarto-html/popper.min.js"></script>
<script src="Manuscript_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Manuscript_files/libs/quarto-html/anchor.min.js"></script>
<link href="Manuscript_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Manuscript_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Manuscript_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Manuscript_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Manuscript_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

<link href="Manuscript_files/libs/table1-1.0/table1_defaults.css" rel="stylesheet">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Wastewater Surveillance for SARS-CoV-2 Data Analysis Project</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Leah Lariscy </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">April 25, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<div class="cell">

</div>
<section id="abstract" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Abstract</h1>
<p>Infection by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), causing COVID-19, is followed by the shedding of viral particles by multiple excretory functions, including stool and urine production. Subsequently, these viral particles can be detected in wastewater influent via RNA extraction, followed by Reverse Transcriptase Quantitative Polymerase Chain Reaction (RT-qPCR) utilizing SARS-CoV-2 specific primers. Wastewater-based detection methods have been utilized across the globe as an independent and parallel indicator of SARS-CoV-2 viral prevalence and, depending on community-specific factors, as a leading indicator of clinical case trends. WBE methods may also become a primary tool in understanding transmission dynamics as case under-reporting increases.</p>
<div style="page-break-after: always;"></div>
</section>
<section id="introduction" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Introduction</h1>
<section id="general-background-information" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="general-background-information"><span class="header-section-number">2.1</span> General Background Information</h2>
<p>At the time of writing it has been three years since the COVID-19 Pandemic was declared, followed by a lockdown across the globe. To date, the World Health Organization reports over 760 million confirmed cases and 6.9 million deaths globally <span class="citation" data-cites="WHOCoronavirusCOVID19">(<a href="#ref-WHOCoronavirusCOVID19" role="doc-biblioref"><em><span>WHO Coronavirus</span> (<span>COVID-19</span>) <span>Dashboard</span></em>, n.d.</a>)</span>. The causative agent, severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), is a positive-sense single-stranded RNA zoonotic virus that infects cells via its spike proteins or “crowns” <span class="citation" data-cites="vkovskiCoronavirusBiologyReplication2021">(<a href="#ref-vkovskiCoronavirusBiologyReplication2021" role="doc-biblioref">V’kovski, Kratzel, Steiner, Stalder, &amp; Thiel, 2021</a>)</span>. When infected by this airborne pathogen, people report experiencing a wide range of mild to severe symptoms, including shortness of breath, cough, head and body aches, and loss of taste or smell. Older adults and others with underlying issues are at higher risk of experiencing more severe outcomes, including death <span class="citation" data-cites="COVID19YourHealth2020">(<a href="#ref-COVID19YourHealth2020" role="doc-biblioref"><span>“<span>COVID-19</span> and <span>Your Health</span>,”</span> 2020</a>)</span>. However, long-term effects of COVID-19 on the body are not well understood yet and it may be years before enough research has been done to well-characterize these impacts. The effects of this pandemic have been far-reaching, with influence practically in every aspect of society, indicating a need to keep a close eye on this infectious disease. However, as the pandemic progresses, clinical reporting continues to dwindle, leaving researchers needing a better means of collecting data on the spread of the virus.</p>
<p>Early on in the pandemic, wastewater-based epidemiology (WBE) was identified as a promising method for detection of SARS-CoV-2 in a population. This is due to the ability of SARS-CoV-2 particles to shed from the body during and after infection through excretory functions, namely stool production <span class="citation" data-cites="wuProlongedPresenceSARSCoV22020">(<a href="#ref-wuProlongedPresenceSARSCoV22020" role="doc-biblioref">Wu et al., 2020</a>)</span>. Viral particles in wastewater are able to be detected via Reverse Transcriptase Quantitative Polymerase Chain Reaction (RT-qPCR) by amplification of universally-conserved gene targets on the SARS-CoV-2 genome <span class="citation" data-cites="ahmedFirstConfirmedDetection2020">(<a href="#ref-ahmedFirstConfirmedDetection2020" role="doc-biblioref">Ahmed et al., 2020</a>)</span>. The use of universal targets allows for the detection of the virus regardless of what variant is present. As of writing, over 4,000 sites worldwide have contributed to wastewater monitoring of SARS-CoV-2 <span class="citation" data-cites="COVIDPoops19">(<a href="#ref-COVIDPoops19" role="doc-biblioref"><em><span>COVIDPoops19</span></em>, n.d.</a>)</span>. Not only is WBE an effective way of detecting viral presence, but quantification of wastewater viral load has been shown to correlate strongly with COVID-19 clinical cases when reporting efforts were strong, indicating the potential for it to be used as an alternative disease monitoring method.</p>
<p>Paragraph 3: Surveillance program background. Aims, County-level parallel analysis of wastewater surveillance data and reported clinical cases</p>
</section>
<section id="description-of-data-and-data-source" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="description-of-data-and-data-source"><span class="header-section-number">2.2</span> Description of data and data source</h2>
<section id="sars-cov-2-wastewater-surveillance-data" class="level3">
<h3 class="anchored" data-anchor-id="sars-cov-2-wastewater-surveillance-data">SARS-CoV-2 wastewater surveillance data</h3>
<p>Sample collection for wastewater surveillance began June 30, 2020 and occurred twice weekly (excluding major holidays) through December 21, 2022. Raw wastewater influent was collected from three Wastewater Treatment Plants in Athens-Clarke County and viral RNA was extracted. RT-qPCR was used to calculate the cycle threshold (Ct) value of each reaction, which is used to estimate SARS-CoV-2 copies per liter of wastewater in the data cleaning process (Section 3.2).</p>
</section>
<section id="wastewater-treatment-plant-data" class="level3">
<h3 class="anchored" data-anchor-id="wastewater-treatment-plant-data">Wastewater treatment plant data</h3>
<p>Wastewater influent flow data, including millions of gallons per day (MGD) and total suspended solids (TSS), were collected for each corresponding sampling date.</p>
</section>
<section id="covid-19-clinical-data" class="level3">
<h3 class="anchored" data-anchor-id="covid-19-clinical-data">COVID-19 clinical data</h3>
<p>COVID-19 clinical data were obtained from the Georgia Department of Public Health website <span class="citation" data-cites="COVID19StatusReport">(<a href="#ref-COVID19StatusReport" role="doc-biblioref"><span>“<span>COVID-19 Status Report</span>,”</span> n.d.</a>)</span> which includes COVID-19 reported clinical case data for the state of Georgia. These data include reported cases, demographics, comorbidities, deaths, and more. However, not all data is necessary for analysis and will be subset to include only symptom onset, positive case, and administered test data for Clarke County. (Section 3.2).</p>
</section>
</section>
<section id="hypotheses-to-be-addressed" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="hypotheses-to-be-addressed"><span class="header-section-number">2.3</span> Hypotheses to be addressed</h2>
<p>This study aims to conduct a parallel analysis between SARS-CoV-2 wastewater surveillance data and reported COVID-19 clinical case data in Athens-Clarke County, Georgia. It is expected that wastewater quantifications and reported case data will have a positive linear relationship. It is also hypothesized that certain wastewater-clinical relationships will be stronger than others, see Table 1 for variables of interest.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="methods" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Methods</h1>
<section id="data-acquisition" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="data-acquisition"><span class="header-section-number">3.1</span> Data acquisition</h2>
<section id="wastewater-surveillance-data" class="level3">
<h3 class="anchored" data-anchor-id="wastewater-surveillance-data">Wastewater surveillance data</h3>
<p>24-hour composite raw wastewater influent samples were collected twice-weekly from three treatment facilities in Clarke County and stored at 4°C until ready for extraction. 280 𝜇L of wastewater in replicates (n=6) was aliquoted and direct RNA extraction (Zymo Research, R2042) was performed on the same day as sample collection.Each extraction replicate was reverse transcribed and amplified using RT-qPCR and a SARS-CoV-2 specific primer-probe panel (IDT, 10006713), done in technical replicates (n=3). As part of the qPCR panel, two gene targets on the N-gene of the SARS-CoV-2 genome were used for amplification. The N1 and N2 gene targets were chosen due to their conservation across variants and act as universal detectors of the virus [@]. Cycle threshold (Ct) values were then calculated for each PCR reaction, which were then used to estimate SARS-CoV-2 viral load (See Section 3.2 for more detail).</p>
</section>
<section id="wastewater-treatment-plant-data-1" class="level3">
<h3 class="anchored" data-anchor-id="wastewater-treatment-plant-data-1">Wastewater treatment plant data</h3>
<p>For each treatment plant and collection date, MGD and TSS was shared by plant operators directly with our team and compiled throughout the study period.</p>
</section>
<section id="covid-19-clinical-data-1" class="level3">
<h3 class="anchored" data-anchor-id="covid-19-clinical-data-1">COVID-19 clinical data</h3>
<p>Reported case and testing data were downloaded from Georgia Department of Public Health’s COVID-19 Status Report Page in January 2023.</p>
</section>
</section>
<section id="data-import-and-cleaning" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="data-import-and-cleaning"><span class="header-section-number">3.2</span> Data import and cleaning</h2>
<section id="calculating-wastewater-viral-load" class="level3">
<h3 class="anchored" data-anchor-id="calculating-wastewater-viral-load">Calculating wastewater viral load</h3>
<p>Amplification standard curve equations where generated from serially diluted synthetic controls (Supplementary Table). This was done for N1 and N2 gene targets for both surveillance years because RT-qPCR methods were altered after the first year. Viral copies per microliter (cp/uL) of each PCR reaction was estimated by transforming the Ct value using the appropriate standard curve equation. For reactions with no detection, concentration was set to the theoretical limit of detection (0.004 cp/uL). Data sets for each standard curve was transformed separately before being combined. After combining all qPCR data, copies per liter (cp/L) of wastewater was calculated for each technical replicate using Equation 1. Average cp/L of wastewater was determined by taking the mean of all technical replicates per sample per gene target.</p>
<p>INSERT EQUATION 1</p>
<p>WWTP data was then transformed by converting the flow rate from millions of gallons per day to liters per day. qPCR data was then combined with WWTP data and total viral load per sample was calculated by multiplying average copies per liter by the flow rate. Average viral load for each sample was then determined by taking the mean of the viral load for both N1 and N2 gene targets. County-level daily viral load was calculated by taking the sum of the average viral load from each of the three WWTP samples.</p>
</section>
<section id="calculating-wastewater-qpcr-assay-positivity" class="level3">
<h3 class="anchored" data-anchor-id="calculating-wastewater-qpcr-assay-positivity">Calculating wastewater qPCR assay positivity</h3>
<p>qPCR assay positivity was determined for each collection date by finding the proportion of assays that detected SARS-CoV-2 out of the total number of assays performed. This includes all biological and technical replicate assays that were performed for a given sampling date. County-level assay positivity data was then combined with county-level viral load data.</p>
</section>
<section id="adding-covid-19-clinical-data" class="level3">
<h3 class="anchored" data-anchor-id="adding-covid-19-clinical-data">Adding COVID-19 clinical data</h3>
<p>Of the downloaded Georgia DPH data, the data set containing positive case counts was retained. Of these data, daily reported case counts and case seven-day moving average were kept for analysis. These data were then subset to include only data from Clarke County collected during the surveillance period and combined with wastewater data.</p>
</section>
</section>
<section id="statistical-analysis" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="statistical-analysis"><span class="header-section-number">3.3</span> Statistical analysis</h2>
<p>Through exploration of time series and wastewater-clinical correlations, prominent relationships were identified for further analysis. Wastewater variables were used to train linear regression models to predict clinical observations. Model performance metrics (RSQ and RMSE) were assessed against each other and against a null model. Cross-validation was used to validate performances. The best model was then determined and used to predict values in a new set of data.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="results" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Results</h1>
<section id="time-series-analysis" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="time-series-analysis"><span class="header-section-number">4.1</span> Time-series analysis</h2>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../figures/exploratory/covariate/log10VL_time.png" class="img-fluid figure-img" width="1094"></p>
<p></p><figcaption class="figure-caption">Figure 1. Wastewater viral load time-series</figcaption><p></p>
</figure>
</div>
</div>
</div>
</section>
<section id="correlation-analysis" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="correlation-analysis"><span class="header-section-number">4.2</span> Correlation analysis</h2>
<p>Two wastewater and two clinical case metrics were identified as features of interest during exploratory analysis steps where stronger wastewater-clinical correlations were observed between these variables than in others. These include wastewater viral load, wastewater assay positivity rate, clinical case seven-day moving average, and test administration positivity rate (Table 1).</p>
<div class="cell">
<div class="cell-output-display">

<div class="Rtable1"><table class="Rtable1">
<thead>
<tr>
<th class="rowlabel firstrow lastrow"></th>
<th class="firstrow lastrow"><span class="stratlabel">Overall<br><span class="stratn">(N=224)</span></span></th>
</tr>
</thead>
<tbody>
<tr>
<td class="rowlabel firstrow">Log10 Viral Load</td>
<td class="firstrow"></td>
</tr>
<tr>
<td class="rowlabel">Mean (SD)</td>
<td>13.7 (0.807)</td>
</tr>
<tr>
<td class="rowlabel lastrow">Median [Min, Max]</td>
<td class="lastrow">13.9 [12.1, 15.2]</td>
</tr>
<tr>
<td class="rowlabel firstrow">Assay Positivity</td>
<td class="firstrow"></td>
</tr>
<tr>
<td class="rowlabel">Mean (SD)</td>
<td>43.8 (32.3)</td>
</tr>
<tr>
<td class="rowlabel lastrow">Median [Min, Max]</td>
<td class="lastrow">39.3 [0, 99.1]</td>
</tr>
<tr>
<td class="rowlabel firstrow">Clinical Case Seven-Day Moving Average</td>
<td class="firstrow"></td>
</tr>
<tr>
<td class="rowlabel">Mean (SD)</td>
<td>11.7 (9.13)</td>
</tr>
<tr>
<td class="rowlabel lastrow">Median [Min, Max]</td>
<td class="lastrow">9.35 [0.200, 34.7]</td>
</tr>
<tr>
<td class="rowlabel firstrow">Daily Reported Clinical Cases</td>
<td class="firstrow"></td>
</tr>
<tr>
<td class="rowlabel">Mean (SD)</td>
<td>44.5 (62.4)</td>
</tr>
<tr>
<td class="rowlabel lastrow">Median [Min, Max]</td>
<td class="lastrow">25.0 [0, 365]</td>
</tr>
<tr>
<td class="rowlabel firstrow">PCR Test Positivity</td>
<td class="firstrow"></td>
</tr>
<tr>
<td class="rowlabel">Mean (SD)</td>
<td>10.3 (9.16)</td>
</tr>
<tr>
<td class="rowlabel lastrow">Median [Min, Max]</td>
<td class="lastrow">7.16 [0, 40.6]</td>
</tr>
</tbody>
</table>
</div>
<p>Table 1. Statistics on clinical and wastewater variables of interest</p>
</div>
</div>
</section>
<section id="model-analysis" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="model-analysis"><span class="header-section-number">4.3</span> Model analysis</h2>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../figures/modeling/assaypos_7dma.png" class="img-fluid figure-img" width="1094"></p>
<p></p><figcaption class="figure-caption">Figure 1. Observed Clinical Cases vs Wastewater Predictions</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="discussion" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Discussion</h1>
<section id="summary-and-interpretation" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="summary-and-interpretation"><span class="header-section-number">5.1</span> Summary and Interpretation</h2>
<p><em>Summarize what you did, what you found and what it means.</em></p>
</section>
<section id="strengths-and-limitations" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="strengths-and-limitations"><span class="header-section-number">5.2</span> Strengths and Limitations</h2>
<p><em>Discuss what you perceive as strengths and limitations of your analysis.</em></p>
</section>
<section id="conclusions" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="conclusions"><span class="header-section-number">5.3</span> Conclusions</h2>
<p><em>What are the main take-home messages?</em></p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="references" class="level1 unnumbered" data-number="6">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">6 References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-ahmedFirstConfirmedDetection2020" class="csl-entry" role="doc-biblioentry">
Ahmed, W., Angel, N., Edson, J., Bibby, K., Bivins, A., O’Brien, J. W., … Mueller, J. F. (2020). First confirmed detection of <span>SARS-CoV-2</span> in untreated wastewater in <span>Australia</span>: <span>A</span> proof of concept for the wastewater surveillance of <span>COVID-19</span> in the community. <em>Science of The Total Environment</em>, <em>728</em>, 138764. <a href="https://doi.org/10.1016/j.scitotenv.2020.138764">https://doi.org/10.1016/j.scitotenv.2020.138764</a>
</div>
<div id="ref-COVID19YourHealth2020" class="csl-entry" role="doc-biblioentry">
<span>COVID-19</span> and <span>Your Health</span>. (2020). https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html.
</div>
<div id="ref-COVID19StatusReport" class="csl-entry" role="doc-biblioentry">
<span>COVID-19 Status Report</span>. (n.d.). https://dph.georgia.gov/covid-19-status-report.
</div>
<div id="ref-COVIDPoops19" class="csl-entry" role="doc-biblioentry">
<em><span>COVIDPoops19</span></em>. (n.d.). https://www.arcgis.com/apps/dashboards/c778145ea5bb4daeb58d31afee389082.
</div>
<div id="ref-vkovskiCoronavirusBiologyReplication2021" class="csl-entry" role="doc-biblioentry">
V’kovski, P., Kratzel, A., Steiner, S., Stalder, H., &amp; Thiel, V. (2021). Coronavirus biology and replication: Implications for <span>SARS-CoV-2</span>. <em>Nature Reviews Microbiology</em>, <em>19</em>(3), 155–170. <a href="https://doi.org/10.1038/s41579-020-00468-6">https://doi.org/10.1038/s41579-020-00468-6</a>
</div>
<div id="ref-WHOCoronavirusCOVID19" class="csl-entry" role="doc-biblioentry">
<em><span>WHO Coronavirus</span> (<span>COVID-19</span>) <span>Dashboard</span></em>. (n.d.). https://covid19.who.int.
</div>
<div id="ref-wuProlongedPresenceSARSCoV22020" class="csl-entry" role="doc-biblioentry">
Wu, Y., Guo, C., Tang, L., Hong, Z., Zhou, J., Dong, X., … Huang, X. (2020). Prolonged presence of <span>SARS-CoV-2</span> viral <span>RNA</span> in faecal samples. <em>The Lancet Gastroenterology &amp; Hepatology</em>, <em>5</em>(5), 434–435. <a href="https://doi.org/10.1016/S2468-1253(20)30083-2">https://doi.org/10.1016/S2468-1253(20)30083-2</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>